jdk/make/CompileLaunchers.gmk

```diff
@@ -1,3 +1,8 @@
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2017, 2019 All Rights Reserved
+# ===========================================================================
```

Comment:
Have you tried to achieve your goal modifying _only_ files within the `closed` folder?

Reply:
I tried, but wasn't successful.  I am open to suggestions, though.